spec/index.bs

```diff
@@ -6713,237 +6731,217 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
 
 ## Texture Format Capabilities ## {#texture-format-caps}
 
-Issue: add multisampling to these tables
+The {{GPUTextureComponentType}} of a texture format is expressed by the
+component type in the format name:
+
+<table class=data>
+    <thead>
+        <tr><th>Format component name <th>{{GPUTextureComponentType}}
+    </thead>
+    <tr><td data-componenttype=float>float <td>{{GPUTextureComponentType/float}}
+    <tr><td data-componenttype=norm>unorm <td>{{GPUTextureComponentType/float}}
+    <tr><td data-componenttype=norm>snorm <td>{{GPUTextureComponentType/float}}
+    <tr><td data-componenttype=int>uint <td>{{GPUTextureComponentType/uint}}
+    <tr><td data-componenttype=int>sint <td>{{GPUTextureComponentType/sint}}
+</table>
+
+Issue: Add multisampling to the tables below.
 
 ### Plain color formats ### {#plain-color-formats}
 
 All plain color formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}}, and {{GPUTextureUsage/SAMPLED}} usage.
 
-Only formats with {{GPUTextureComponentType/float|GPUTextureComponentType."float"}} can be blended.
+Only formats with {{GPUTextureComponentType}} {{GPUTextureComponentType/"float"}} can be blended.
```

Comment:
why is this change?

Reply:
I didn't think it made sense to use the `.` syntax here since "float" isn't a member of GPUTextureComponentType. Though I'd be fine with adopting that style now that I think about it.